### ISSUE TITLE
Add POUR Headings

### ISSUE THREAD

**Issue by mgifford** [GSA/openacr#264]
The WCAG A, AA & AAA should be organized under POUR titles.

**Comment by dmundra** [comment]
I build the fields from the catalog and this would be easier if we store the POUR sections in the catalog. Not required but wanted to share that. If we didn't want to add it to the catalog then one idea would be store a data structure matching criteria to their pour heading. Should we add this to the catalog?

**Comment by mgifford** [comment]
So much about WCAG is tied to POUR. I expect that there will be other advantages if we update the catalog to contain this structure. Thanks!

**Comment by bruce-usab** [comment]
I agree that the catalog reflecting the POUR structure is useful.

One caveat though: My perspective is that POUR is not terribly meaningful for U.S. federal employees consuming VPAT-driven ACRs.  So I would suggest not leaning on POUR much more than as a hierarchy for the WCAG SC numbering.

**Comment by mgifford** [comment]
That is interesting. I guess before the refresh WCAG's POUR framework wasn't very relevant inside of the USA. When did the VPATs move to supporting A, AA & AAA structures? I find POUR handy just to help understand some structure, but this is another instance where a Word based table structure doesn't really align with adding additional context. 

Agreed with your direction @bruce-usab - thanks!

**Comment by bruce-usab** [comment]
> When did the VPATs move to supporting A, AA & AAA structures?

Just going from memory here, but I think that was pretty soon after the SC were incorporated explicitly into the VPAT.  If memory serves, at one point "WCAG 2.0 Level AA conformant" was a single yes/no checkbox.
